### DISCUSSION TITLE
Parallel implementation

### DISCUSSION THREAD

**Discussion by jdellag** [lululxvi/deepxde#1104] [Q&A]
I have looked everywhere but can not seem to find any mention of the ability to use this library in a distributed HPC setting (i.e. utilizing more than 1 node in such an environment). Due to this, I am convinced that this is not possible. Can someone please confirm?

**Comment by lululxvi** [comment]
https://github.com/lululxvi/deepxde/pull/703 https://github.com/lululxvi/deepxde/pull/1094
